tests/test_register_schema.py

```diff
@@ -0,0 +1,130 @@
+from contextlib import redirect_stderr
+import json
+import jsonschema
+import logging
+import pytest
+import io
+
+from jupyter_telemetry.eventlog import EventLog
+
+
+def test_register_invalid():
+    """
+    Test registering invalid schemas fails
+    """
+    el = EventLog()
+    with pytest.raises(jsonschema.SchemaError):
+        el.register_schema({
+            # Totally invalid
+            'properties': True
+        })
+
+    with pytest.raises(ValueError):
+        el.register_schema({
+            'properties': {}
+        })
+
+    with pytest.raises(ValueError):
```

Comment:
It's not immediately obvious to me why this should fail. I certainly didn't notice that the failure is caused by an extra `$` before `version`. Maybe add a comment to this source code saying 
```
# testing invalid key: "$version" when it should be "version".
```

Reply:
Done!